test/validation/api/pool/pool.c

```diff
@@ -1480,6 +1480,127 @@ static void test_packet_pool_ext_alloc_seg(void)
 	packet_pool_ext_alloc(PKT_LEN_SEGMENTED);
 }
 
+static void test_packet_pool_ext_disassemble(void)
+{
+	odp_shm_t shm;
+	odp_pool_t pool;
+	odp_pool_ext_param_t param;
+	odp_pool_ext_capability_t capa;
+	uint32_t i, j, buf_size, num_buf, num_pkt, num_alloc, buf_index;
+	uint32_t pkt_len, head_offset, trailer_size, headroom, max_headroom;
+	uint32_t hr, max_payload, buf_data_size;
+	uint32_t num_seg;
+	void *buf[EXT_NUM_BUF];
+	odp_packet_t pkt_tbl[EXT_NUM_BUF];
+
+	CU_ASSERT_FATAL(odp_pool_ext_capability(ODP_POOL_PACKET, &capa) == 0);
+
+	if (capa.pkt.max_segs_per_pkt < 2)
+		return;
```

Comment:
The test could be made conditional based on this.

Reply:
Fixed in v15